frontend/ic_vetkeys/src/utils/utils.ts

```diff
@@ -538,38 +559,65 @@ const DERIVED_KEY_MATERIAL_NONCE_LENGTH = 12;
  */
 export class DerivedKeyMaterial {
     readonly #hkdf: CryptoKey;
+    readonly #raw: CryptoKey;
 
     /**
      * @internal constructor
      */
-    private constructor(cryptokey: CryptoKey) {
-        this.#hkdf = cryptokey;
+    private constructor(hkdf: CryptoKey, raw: CryptoKey) {
+        this.#hkdf = hkdf;
+        this.#raw = raw;
     }
 
-    static fromCryptoKey(cryptokey: CryptoKey): DerivedKeyMaterial {
-        return new DerivedKeyMaterial(cryptokey);
+    static async fromCryptoKey(raw: CryptoKey): Promise<DerivedKeyMaterial> {
```

Comment:
This looks like a breaking change.
My understanding is that if someone has stored the CryptoKey returned from DerivedKeyMaterial::getCryptoKey with the current library version in an IndexedDB, then passing that to DerivedKeyMaterial::fromCryptoKey with the new library version won't work.

Not sure yet what the best way is to fix it. Do we also have to make this backward-compatible?